code/game/machinery/fabricators/modular_fabricator.dm

```diff
@@ -149,11 +149,12 @@
 				categories_associative[cat] = list()
 
 			//Calculate cost
+
 			var/list/material_cost = list()
 			for(var/material_id in D.materials)
 				material_cost += list(list(
 					"name" = material_id,
-					"amount" = ( D.materials[material_id] / MINERAL_MATERIAL_AMOUNT) * creation_efficiency,
+					"amount" = ( D.materials[material_id] / (MINERAL_MATERIAL_AMOUNT / 2)) * creation_efficiency,
```

Comment:
Why is this dividing by 2?

Reply:
Because due to what I think is an oversight that value shouldn't be 2000 but instead 1000.
As tonty said using "magic numbers is bad" so I divided it by two.
This is what ensures the late will consume 160% materials as described and as you can see in the photos I provided. Sadly I'm not at home and can't look at the code again but I can probably explain exactly why and the math behind it Monday!

Reply:
It just seems a little strange to me. Materials should always be MINERAL_MATERIAL_AMOUNT and if they aren't then the oversight should be fixed.

Reply:
For some reason both with auto and protolathes everything seems to always double. Protolathes do the material cost calculation twice, once for each capacitor, this leads to materials always costing half of what value you assign them in code.
I will look into it Monday and see if I can fix it properly. But I'll tell you. I thought of doing this exact thing of dividing by two in protolathes as well in the future...

Reply:
Ok the issue goes a bit deeper, since Mech fabricators share /modular_fabricator/ with autolathes, without the change, the calculation works as intended for Mech Fabs but not Autolathes, with the change, it does the other way, I will try to find a way to make everything be as it should